main.py

```diff
@@ -1,18 +1,16 @@
-from router import auth
+from router import auth,session
```

Comment:
:o how is this even working?
The name of the file is `sessions.py` but you're importing `session`?
This should clearly give an error, something like, `session` is an unknown import symbol or something like that?
how is the code getting to know that when you import `session` from router module, it needs to look into `sessions.py` file?
Because the naming should be exactly same right?

Reply:
The file name is `session.py` only, no.